Traduire en NL et ES les nouveaux éléments (PIX-17067)

pix-service-prod opened the pull request:
Exported the most recent locale files from Phrase

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr11781.review.pix.fr)
  * [App (.org)](https://app-pr11781.review.pix.org)
  * [Orga (.fr)](https://orga-pr11781.review.pix.fr)
  * [Orga (.org)](https://orga-pr11781.review.pix.org)
  * [Certif (.fr)](https://certif-pr11781.review.pix.fr)
  * [Certif (.org)](https://certif-pr11781.review.pix.org)
  * [Junior](https://junior-pr11781.review.pix.fr)
  * [Admin](https://admin-pr11781.review.pix.fr)
  * [API](https://api-pr11781.review.pix.fr/api/)
  * [API MaDDo](https://pix-api-maddo-review-pr11781.osc-fr1.scalingo.io/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr11781.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr11781/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr11781/environment)
  * [scalingo api-maddo](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-maddo-review-pr11781/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr11781/environment)

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

yannbertrand commented on mon-pix/translations/es.json:
```diff
@@ -57,7 +57,7 @@
       "internal-server-error": "Se ha producido un error interno. Nuestros equipos están resolviendo el problema. Vuelve a intentarlo más tarde.",
       "login-unauthorized-error": "La dirección de correo electrónico o el nombre de usuario y/o la contraseña introducidos son incorrectos.",
-      "login-unexpected-error": "No se ha podido conectar. Por favor, inténtelo de nuevo en unos momentos. Si el problema persiste, '<'a href=\"{supportHomeUrl}\" target=\"blank\"'>'póngase en contacto con nosotros'</a>'.",
+      "login-unexpected-error": "No se ha podido conectar. Por favor, inténtelo de nuevo en unos momentos. Si el problema persiste, '<'a href=\"https://pix.fr/support\" target=\"blank\"'>'póngase en contacto con nosotros'</a>'.",
```
🤔

jujumathieu commented:
Tout semble bon côté devcomp (rien de modifié côté contenu)

AndreiaPena commented on mon-pix/translations/es.json:
```diff
@@ -1330,6 +1336,8 @@
         "missing-code": "Por favor, introduce un código.",
         "not-found": "Tu código es incorrecto, comprueba el formato (por ejemplo, ABCDEF234) o ponte en contacto con el organizador."
       },
+      "explanation-link": "https://support.pix.org/fr/support/solutions/articles/15000029147-qu-est-ce-qu-un-code-parcours-et-comment-l-utiliser-",
+      "explanation-message": "¿Qué es un código de test y cómo se utiliza?",
```
je n'ai pas trouvé ces clés en `fr/en` (ça ne concerne pas certif, donc je laisse la team owner s'en charger)

Faraopix replied:
Elles ont été supprimées récemment en fr/en

AndreiaPena commented on mon-pix/translations/nl.json:
```diff
@@ -1357,8 +1365,8 @@
       "first-title": "Een Pix-certificaat controleren",
-      "label": "Verificatiecode",
-      "sub-label": "voorbeeld: P-XXXXXXXX",
+      "label": "Verificatiecode (P-XXXXXXXX)",
```
```suggestion
      "label": "Verificatiecode",
```

Agnes-V replied:
Done

AndreiaPena commented on mon-pix/translations/es.json:
```diff
@@ -1357,7 +1365,7 @@
       },
       "first-title": "Comprobar un certificado Pix",
-      "label": "Código de comprobación",
+      "label": "Código de comprobación (P-XXXXXXXX)",
```
```suggestion
      "label": "Código de comprobación",
```

Agnes-V replied:
Done

AndreiaPena commented on mon-pix/translations/nl.json:
```diff
@@ -1939,6 +1947,7 @@
       "badges-title": "Je thematische resultaten",
       "details": {
         "caption": "Details van je resultaten in percentages volgens vaardigheden",
+        "flash-pix-score": "{score, number, ::.} Pix",
```
ne doit pas être ajouté

Agnes-V replied:
Done

AndreiaPena commented on mon-pix/translations/nl.json:
```diff
@@ -1948,6 +1957,11 @@
       },
       "disabled-share": "Deze route is gedeactiveerd door de organisator.'<br>'Het is niet langer mogelijk om resultaten te versturen.",
       "error": "Oeps, er is een fout gemaakt!",
+      "flash": {
+        "abstract": "Goed gedaan, je bent er helemaal voor gegaan!",
+        "nonDefinitive": "(berekening niet definitief)",
+        "pixScore": "Je hebt {score, number, ::.} behaald. Pix"
+      },
```
ne doit pas être ajouté

Agnes-V replied:
Done

AndreiaPena commented on mon-pix/translations/es.json:
```diff
@@ -1938,6 +1947,7 @@
       "badges-title": "Tus resultados temáticos",
       "details": {
         "caption": "Detalle de tus resultados en términos porcentuales según las competencias",
+        "flash-pix-score": "{score, number, ::.} Pix",
```
ne doit pas être ajouté

Agnes-V replied:
Done

AndreiaPena commented on mon-pix/translations/es.json:
```diff
@@ -1947,6 +1957,11 @@
+        "abstract": "¡Enhorabuena, has llegado hasta el final!",
+        "nonDefinitive": "(cálculo no definitivo)",
+        "pixScore": "Has obtenido {score, number, ::.} Pix"
+      },
```
ne doit pas être ajouté

Agnes-V replied:
Done

AndreiaPena commented on mon-pix/translations/nl.json:
```diff
@@ -1989,7 +2003,8 @@
         },
-        "notification": "Als je de mislukte vragen opnieuw probeert of alles op nul zet om alles opnieuw te proberen, kan je organisatie je laatste resultaat niet meer zien. Om je resultaat mee te laten tellen door de organisator van de test, moet je het opnieuw insturen."
+        "notification": "Als je de mislukte vragen opnieuw maakt of op nul zet om het opnieuw te proberen, moet je je resultaten opnieuw opsturen zodat ze kunnen worden geteld door de organisator. De organisator blijft toegang houden tot je eerder ingestuurde resultaten.",
+        "notifications": "Als je de mislukte vragen opnieuw probeert of alles op nul zet om alles opnieuw te proberen, kan je organisatie je laatste resultaat niet meer zien. Om je resultaat mee te laten tellen door de organisator van de test, moet je het opnieuw insturen."
```
supprimer la clé au pluriel, garder la clé au singulier (à voir quel contenu garder en revanche)

(ne concerne pas certif, donc je laisse la team owner s'en charger)

Alexandre-Monney replied:
Le contenu a garder est celui du singulier, on y a touché récemment (https://github.com/1024pix/pix/pull/11666/commits/edcad27cba9910bd98dd3cac699f5dd96e32ef68#diff-005112505ed6804aa6f4e2267a9004af90ced1a3fa341246a2fa4553f63add44)

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

Agnes-V commented:
Vérif done et export des modif faites côté Certif, merci encore @AndreiaPena ✨

Alexandre-Monney commented on mon-pix/translations/es.json:
```diff
@@ -261,6 +262,9 @@
         "computing-description": "Un problema técnico nos ha impedido informarle de sus resultados. Por favor, conéctese de nuevo mañana en Pix, en la sección \"Mi curso\", para comprobar sus resultados. Le pedimos disculpas por las molestias ocasionadas.",
         "not-obtained": "Certificado no obtenido :",
         "obtained": "Certificado obtenido",
+        "SIXTH_GRADE": {
+          "title": "Conciencia digital"
+        },
```
A été remplacé par juste en dessous title.digital-awarness

Alexandre-Monney commented on mon-pix/translations/es.json:
```diff
@@ -1988,7 +1997,8 @@
           "warning-text": "Tus Pix, niveles y certificabilidad obtenidos durante este test se van a eliminar."
         },
-        "notification": "Si vuelves a intentar las preguntas fallidas o pones a cero para volver a intentarlo todo, tu organización ya no podrá ver tu último resultado. Para que el organizador de la prueba contabilice tu resultado, tendrás que volver a enviarlo."
+        "notification": "Si vuelve a realizar las preguntas fallidas o se pone a cero para intentarlo de nuevo, tendrá que enviar de nuevo sus resultados para que puedan ser contabilizados por el organizador. El organizador seguirá teniendo acceso a tus resultados enviados anteriormente.",
+        "notifications": "Si vuelves a intentar las preguntas fallidas o pones a cero para volver a intentarlo todo, tu organización ya no podrá ver tu último resultado. Para que el organizador de la prueba contabilice tu resultado, tendrás que volver a enviarlo."
```
Ne garder que celle au singulier
(voir https://github.com/1024pix/pix/pull/11666/commits/edcad27cba9910bd98dd3cac699f5dd96e32ef68#diff-005112505ed6804aa6f4e2267a9004af90ced1a3fa341246a2fa4553f63add44)

Alexandre-Monney commented on mon-pix/translations/es.json:
```diff
@@ -2040,7 +2050,7 @@
     },
     "terms-of-service": {
-      "cgu": "Acepto las '<a href=\"https://pix.fr/conditions-generales-d-utilisation\" class=\"link\" target=\"_blank\">'condiciones de uso de Pix'</a>'",
+      "cgu": "Acepto las '<a href=\"https://pix.org/en/terms-and-conditions\" class=\"link\" target=\"_blank\">'condiciones de uso de Pix'</a>'",
```
ça redirige vers la page /en/ ?

Alexandre-Monney commented on mon-pix/translations/nl.json:
```diff
@@ -261,6 +262,9 @@
         "computing-description": "Door een technisch probleem kunnen we je niet informeren over je resultaten. Log morgen opnieuw in op Pix, in de 'Mijn cursus' sectie, om je resultaten te bekijken. Onze excuses voor het ongemak.",
         "not-obtained": "Certificaat niet verkregen :",
         "obtained": "Verkregen certificaat",
+        "SIXTH_GRADE": {
+          "title": "Digitaal bewustzijn"
+        },
```
Comme en ES, remplacé par en dessous title.digital-awarness

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

jujumathieu commented:
On ferme cette PR car il y a actuellement trop de soucis avec la synchro Phrase. On passe donc manuellement dans les fichiers.
